smoke-test/cluster_manager_smoke_test.js

```diff
@@ -15,10 +15,10 @@
 'use strict';
 
 describe('ClusterManagerSmokeTest', () => {
-  if (!process.env.SMOKE_TEST_PROJECT) {
-    throw new Error('Usage: SMOKE_TEST_PROJECT=<project_id> node #{$0}');
+  if (!process.env.GCLOUD_PROJECT) {
```

Comment:
I think we can probably remove this check altogether, because ADC will automatically pick up the `project_id` from the JSON key file.

Reply:
It helps when running the test locally.